### PR TITLE
Fix/arrows

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFLSBP-319

#### Description
This PR makes the usage of arrows on cards consistent and introduces an arrow-button class for when arrows are inside buttons to help this goal.

#### Screenshot of the result
n/a

#### Additional comments or questions
n/a

### CODE REVIEW

File: src/stories/Library/Lists/list-reservation/ListReservationsAndLoans.tsx [hunk 1]
```diff
@@ -103,7 +103,11 @@ export const ListReservationsAndLoans = (
               <p className="text-body-medium-medium">Gebyr {fee},-</p>
             </div>
           ) : null}
-          <button type="button" className="arrow-button">
+          <button
+            type="button"
+            className="arrow-button"
+            aria-label="see reservation"
```
Comment (spaceo) [inline src/stories/Library/Lists/list-reservation/ListReservationsAndLoans.tsx:109]: Adam Antal cinq points, Adam Antal five points!

**Review by spaceo — APPROVED** [review]
LGTM 👍

File: src/stories/Library/Arrows/arrows.scss [hunk 2]
```diff
@@ -42,3 +42,9 @@
 .arrow__link {
   text-decoration: none;
 }
+
+.arrow-button {
+  // This width mimics what the expanded svg width. The class is intended to be
+  // used when the arrow needs to be within a button so they look the same.
+  width: 101px;
```
Comment (JacobArrow) [inline src/stories/Library/Arrows/arrows.scss:49]: How is this handled on smaller screens? Is there room enough for this much space?

Comment (Adamik10) [reply src/stories/Library/Arrows/arrows.scss:49]: on smaller screens the card element should be full width, so yes - there is enough space

**Pull request merged**